bridgecrew_sns_resource.tf

```diff
@@ -1,4 +1,4 @@
-data template_file "message" {
+data "template_file" "message" {
```

Comment:
@JamesWoolfenden the quotes you added here are not inline with the updated tf convention

Reply:
they come from using tf fmt.